packages/core/components/ExternalInput/styles.module.css

```diff
@@ -104,39 +142,35 @@
 }
 
 .ExternalInputModal-thead {
+  background-color: var(--puck-color-white);
   position: sticky;
   top: 0;
   z-index: 1;
 }
 
 .ExternalInputModal-th {
   border-bottom: 1px solid var(--puck-color-grey-09);
-  border-top: 1px solid var(--puck-color-grey-09);
-  font-weight: 700;
+  color: var(--puck-color-grey-04);
+  font-weight: 500;
+  font-size: 14px;
   padding: 16px 24px;
-  opacity: 0.9;
 }
 
 .ExternalInputModal-td {
+  border-bottom: 1px solid var(--puck-color-grey-10);
   padding: 16px 24px;
 }
 
-.ExternalInputModal-tr:nth-of-type(n) {
-  background-color: var(--puck-color-white);
-}
-
-.ExternalInputModal-tr:nth-of-type(2n) {
-  background-color: var(--puck-color-grey-12);
-}
-
 .ExternalInputModal-tr .ExternalInputModal-td:first-of-type {
   font-weight: 500;
+  width: 1%; /* Prevent growing */
+  white-space: nowrap; /* Prevent growing */
 }
 
 @media (hover: hover) and (pointer: fine) {
   .ExternalInputModal-tbody .ExternalInputModal-tr:hover {
-    background: var(--puck-color-grey-11);
-    color: var(--puck-color-azure-04);
+    background: var(--puck-color-azure-11);
+    color: var(--puck-color-azure-05);
```

Comment:
Foreground color needs to be `azure-04` for sufficent contrast. The `04` level is always the foreground text color in light more. Also `azure-12` is the default hover background color in light mode.